lib/MixedModels/LMM.rb

```diff
@@ -311,6 +359,7 @@ def LMM.from_daru(response:, fixed_effects:, random_effects:, grouping:, data:,
       # replace the categorical variable name in (non-interaction) random_effects 
       random_effects.each_index do |i|
         ind = random_effects[i].find_index(name)
+        #?:
```

Comment:
Haha, sorry I forgot to include this comment, I just thought the notation `[ind..ind]` looked strange